Dockerfile

```diff
@@ -7,12 +7,20 @@ WORKDIR /srv/node
 
 RUN buildDeps=' \
     git \
+    yarn \
     ' && \
+    # `apt-transport-https` is required to use https deb repositories
+    apt-get update -y && \
+    apt-get install -y --no-install-recommends apt-transport-https && \
```

Comment:
`apt-transport-https` should be moved to the `buildDeps` list up above. After that, you won't need this duplicate `apt-get install` command. Unless ... does this need to be installed before yarn?